CONTRIBUTING.md

```diff
@@ -75,16 +75,20 @@ In your local repository:
   as previous section requires. You may find this
   [Re-sync Fork With Upstream guide](https://gist.github.com/OleksiyRudenko/8b3ddb664308de0634b53c525e551d8b)
   helpful to fix the issue.
-- ask your peers for review in
-  [FE p2p chat](https://t.me/joinchat/???????????????),
+
+
+### Codereview
+- you will **require** approvals* from **at least two peers**, so seek
+for sufficient support. 
+  - ask your peers for review in [FE Students chat](https://web.telegram.org/#/im?p=s1495296464_549300875144563897),
   post a link to your PR as well
+  - if for some reason there are no reviewers from p2p course available, you may ask your friends from outside the course for a review
+- **you have to provide reviews** on the same task to **at least 2 PRs** of other students of p2p course once your PR is merged, using your learnings from merged PR code review
+- after finishing previous steps ask mentors for review in [FE Questionarium chat](https://web.telegram.org/#/im?p=s1495296464_549300875144563897)
+- once the code review phase successfully finished and you have approval from other students, post a message in [FE Students chat](https://web.telegram.org/#/im?p=s1495296464_549300875144563897): { Here must be the name of your task } — #p2p-pr-done and add the link to your PR. This step is important, as it helps mentors to track your progress!
 
-You will require approvals from at least two peers, so seek
-for sufficient support.
+*Code reviews that are only about code style/code formatting, variables and function naming, introducing semantic constants do not count.*
```

Comment:
`Code reviews that are only about code style/code formatting, variables and function naming, introducing semantic constants do not count.` I prefer to drop this line, because... well, sometimes there is nothing more to change in PR.

Reply:
But if there is this is a clear way to cheat around )
Let them ask us if there is really nothing to do.